rfc/rfc011-verifiable-credential.md

```diff
@@ -0,0 +1,215 @@
+- create `jws` as `header + ".." + base64_rawurlencode(sig)`
+- place the result in the `jws` field of the `proof`:
+
+```json
+{
+  "type": "JsonWebSignature2020",
+  "proofPurpose": "assertionMethod",
+  "verificationMethod": "did:nuts:EgFjg8zqN6eN3oiKtSvmUucao4VF18m2Q9fftAeANTBd#twlH6rB8ArZrknmBRWLXhao3FutZtvOm0hnNhcruenI",
+  "created": "2021-03-09T11:44:56.382202+01:00",
+  "jws": "eyJhbGciOiJFUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..ZXlKaGJHY2lPaUpGVXpJMU5pSjkuVHVqZndMVVJwcnUzbjhuZklhODB1M1M0LW9LcWY0WUs5S2hoZEktUkZPSzdlbnZJTTdLN1E5SzBSeHhRSzNIVWJPTUJyLVlZX1g0eW1YR0pXOHF4UkEuN0F4a3lZekNXTElPZ2Q5TlpnR3p2aHd2UzZZQ3FpRTRPX3FwWGVOSEN6X091S1c0TmJsWkJueTBkZVhXT0lXZ3JNczF4OTZlNmtnaGZGYTRNd0J3TlE="
+}
+```
+
+### 3.2 Content-type
+
+All VCs have a content-type equal to `application/vc+json` when published on the network.
+All VCs MUST be handled as normal JSON. All issued VCs MUST add the `https://nuts.nl/credentials/v1` context.
+VCs within Nuts MAY NOT specify more than one additional type next to `VerifiableCredential`.
+
+### 3.3 Updates
+
+VCs are not updatable, an update can be performed by revoking the current and issuing a new VC.
+
+### 3.4 Identifiers
+
+VC identifiers MUST be constructed as `DID#id` where `id` is unique for the given issuer.
+
+### 3.5 VC Example
+
+Below is an example of a credential.
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://nuts.nl/credentials/v1"
+  ],
+  "type": [
+    "NutsOrganizationCredential",
+    "VerifiableCredential"
+  ],
+  "id": "did:nuts:t1DVVAs5fmNba8fdKoTSQNtiGcH49vicrkjZW2KRqpv#6f91673b-afa9-4d26-9e0f-00d989943275",
+  "issuanceDate": "2021-03-15T16:34:17.687862+01:00",
+  "issuer": "did:nuts:t1DVVAs5fmNba8fdKoTSQNtiGcH49vicrkjZW2KRqpv",
+  "credentialSubject": {
+    "id": "did:nuts:t1DVVAs5fmNba8fdKoTSQNtiGcH49vicrkjZW2KRqpv",
+    "organization": {
+      "city": "IJbergen",
+      "name": "Because we care B.V."
+    }
+  },
+  "proof": {
+    "type": "JsonWebSignature2020",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:nuts:t1DVVAs5fmNba8fdKoTSQNtiGcH49vicrkjZW2KRqpv#h22vbXHX7-lRd1qAJnU63liaehb9sAoBS7RavhvfgR8",
+    "created": "2021-03-15T16:34:17.687862+01:00",
+    "jws": "eyJhbGciOiJFUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..hKcboC8m6YnZPi6ReJAYs0J0Ztn5nxcx2EavoXdtrkWxmE1JZmImW89_8IIgjvfI8XtGeDlEnGywAuY2u7y9Bw"
+  }
+}
+```
+
+## 4. Required chapters for a Verifiable Credential type
+
+All the following chapters MUST be present in the specification of a VC.
+
+### 4.1 CredentialSubject
+
+It MUST specify the contents of the `credentialSubject` JSON field. It MUST specify which parts of the `credentialSubject` are mandatory or optional and the format.
+
+### 4.2 Issuance & distribution
+
+A VC specification MUST specify how a VC is issued and who may issue it. It MUST specify which protocols and standards are used in obtaining the VC. 
```

Comment:
"It MUST specify which protocols and standards are used in obtaining the VC" isn't this RFC the base RFC for all Nuts VC RFC's? Then the protocols and standards are basically what Nuts provides?

Reply:
second time this line is discussed in a PR, too confusing, will remove.